jupyter_releaser/util.py

```diff
@@ -269,18 +271,21 @@ def retry(cmd, **kwargs):
 
 def read_config():
     """Read the jupyter-releaser config data"""
+    config = {}
     if JUPYTER_RELEASER_CONFIG.exists():
-        return toml.loads(JUPYTER_RELEASER_CONFIG.read_text(encoding="utf-8"))
+        config = toml.loads(JUPYTER_RELEASER_CONFIG.read_text(encoding="utf-8"))
 
-    if PYPROJECT.exists():
+    elif PYPROJECT.exists():
         data = toml.loads(PYPROJECT.read_text(encoding="utf-8"))
-        config = data.get("tool", {}).get("jupyter-releaser")
-        if config:
-            return config
+        config = data.get("tool", {}).get("jupyter-releaser") or {}
 
-    if PACKAGE_JSON.exists():
+    elif PACKAGE_JSON.exists():
```

Comment:
Wondering whether this breaks existing repos that have both a `pyproject.toml` and `package.json` file? Previously both files would be checked for the configuration, while this doesn't seem to be the case anymore.

This is for example the case in retrolab or the JupyterLab extension cookiecutter:
- https://github.com/jupyterlab/extension-cookiecutter-ts/blob/3ec170c6c40602d509c98fc6fad46e7267beef60/%7B%7Bcookiecutter.python_name%7D%7D/package.json#L88-L95
- https://github.com/jupyterlab/retrolab/blob/9e534b81be7b71218c105e199bea8e90ea2e383b/package.json#L74-L84

Reply:
Probably we should keep the previous behavior of cascading through the files to be able to pick up the config.

Reply:
Noticed as the check release workflow started to fail here: https://github.com/jupyterlab/retrolab/runs/3535195754

Which seems to be related to the hooks not being run.